packages/superset-ui-core/test/utils/logging.test.ts

```diff
@@ -36,9 +40,16 @@ describe('logging', () => {
     expect(() => {
       logging.error('error');
     }).toThrow('error');
+
+    // to support: npx jest --silent
+    const spy = jest.spyOn(logging, 'trace');
+    spy.mockImplementation(() => {
```

Comment:
Should we include `logging.trace`'s argument here? Like:
`throw new Error('Trace: ' + traceMessage)`

Reply:
This mock just for `logging.trace()`, Ensure that when call the `logging.trace()`, there is `Trace:` at the beginning of the return.